packages/reaction-collections/server/publications/products.js

```diff
@@ -7,12 +7,31 @@
 Meteor.publish("Products", function (productScrollLimit, productFilters) {
   check(productScrollLimit, Match.OneOf(null, undefined, Number));
```

Comment:
I already tried this and found, that we can not completely substitute check() with SS validator, because check package will throw error "not all arguments of publication are checked with check()".
And I like more verbose schema in my case, because it shows full structure of productFilters argument. Don't mistake ValidatedMethod with publication function. You are offering use the same strategy and imho this is right way, but there is no ValidatedPublication package for now, read [here](https://forums.meteor.com/t/package-idea-validated-publication/15516)